externs/browser/html5.js

```diff
@@ -3100,9 +3100,43 @@ WebSocket.prototype.binaryType;
 // HTML5 History
 /**
  * @constructor
+ * @see http://w3c.github.io/html/browsers.html#the-history-interface
  */
 function History() {}
 
+/**
+ * Goes back one step in the joint session history.
+ * If there is no previous page, does nothing.
+ *
+ * @return {undefined}
+ */
+History.prototype.back = function(opt_distance) {};
+
+/**
+ * Goes forward one step in the joint session history.
+ * If there is no next page, does nothing.
+ *
+ * @return {undefined}
+ */
+History.prototype.forward = function() {};
+
+/**
+ * Returns the number of entries in the joint session history.
```

Comment:
Suggested rephrasing: `The number of entries in the joint session history.`, as this is not a function.

Reply:
Done and force pushed the change. 

FWIW the text was an exact copy of the comments from the spec but it does make sense to use this terminology.